source/whats-new.txt

```diff
@@ -44,6 +45,40 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.2:
+
+What's New in 6.2
+-----------------
+
+The {+driver-short+} v6.2 release includes the following features:
+
+- Updates the ``bson`` package version to 6.2.0. This version includes
```

Comment:
Question:
I think the bson package version is pegged to the Node.js version (but can be released prior) -- do you know whether that's true? If so, I think mentioning this as a separate sentence could make readers think that the versions differ.

The 6.1 entry also mentions the package release but combines it with what feature was released.

No changes necessary. I just thought it could be something that might be good to standardize now that the bson package has been separated.